data/_ui/tooltips.txt

```diff
@@ -964,67 +964,67 @@ tip "firing fuel / shot:"
+	`Reduces the ship's turn rate, acceleration, and top speed. Slowing wears off over time.`
 
 tip "firing disruption / shot:"
-	`Each shot disrupts the ship's shields, allowing hostile weapon damage to "leak through" to the hull even if shields are up. Shield disruption wears off over time, but imparting disruption on yourself does not lose effectiveness as your shields become more leaky.`
+	`Allows weapon damage to "leak through" to the ship's hull even if shields are up. Shield disruption wears off over time, but imparting disruption on yourself does not lose effectiveness as your shields become more leaky.`
 
 tip "firing discharge / shot:"
-	`Each shot applies discharge to the ship, reducing its shields over time. Shield discharge wears off over time.`
+	`Shield damage dealt to your ship over time by this weapon per shot.`
 
 tip "firing corrosion / shot:"
-	`Each shot applies corrosion to the ship, reducing the ship's hull over time. Corrosion wears off over time.`
+	`Hull damage dealt to your ship over time by this weapon per shot.`
 
 tip "firing leak / shot:"
-	`Each shot causes the ship's fuel to leak. Fuel leakage wears off over time.`
+	`Fuel damage dealt to your ship over time per shot.`
 
 tip "firing burn / shot:"
-	`Each shot causes the ship to burn, increases the ship's heat over time. Burn wears off over time.`
+	`Heat damage dealt to your ship over time per shot.`
 
 tip "% firing energy / shot:"
-	`Each shot reduces the ship's energy as a percentage of the ship's total energy capacity by this amount.`
+	`Energy consumed per shot, as a percentage of the ship's total energy capacity.`
 
 tip "% firing heat / shot:"
-	`Each shot fired increases the ship's heat as a percentage of the ship's maximum heat capacity by this amount.`
+	`Heat generated per shot, as a percentage of the ship's maximum heat capacity.`
 
 tip "% firing fuel / shot:"
-	`The amount of fuel used or lost per shot fired as a percentage of the ship's total fuel.`
+	`Fuel used or lost per shot, as a percentage of the ship's total fuel.`
 
 tip "% firing hull / shot:"
-	`Each shot fired by this weapon causes this amount of hull damage as a percentage of the ship's total hull.`
+	`Hull damage dealt to your ship per shot, as a percentage of your total hull strength.`
 
 tip "% firing shields / shot:"
-	`Each shot fired by this weapon causes this amount of shield loss as a percentage of the ship's total shields.`
+	`Shield damage dealt to your ship per shot, as a percentage of your total shield strength.`
 
 tip "inaccuracy:"
-	`Maximum inaccuracy of this weapon, in degrees.`
+	`Maximum inaccuracy of this weapon, in degrees. A value of one means the weapon's aim will be at most one degree off-target.`
 
 tip "blast radius:"
-	`When this weapon hits something, everything within this distance (including friendly ships) will be damaged.`
+	`Everything within this distance (including friendly ships) will be damaged when the weapon hits something.`
```

Comment:
This reminds me: looks like we don't have a tooltip for "safe".

Reply:
We also don't have tooltips for "phasing", "gravitational" and probably a lot of other weapon attributes. I don't know which ones are displayed in the interface, though.

Reply:
Just realized these are all weapon attributes, which don't appear unless we explicitly allow them to (which is the opposite of outfit attributes which appear by default unless we hide them), and these don't appear to be given tooltips.